thunder/executors/nvfuserex_impl.py

```diff
@@ -3188,6 +3188,36 @@ def argsort_transform(
 # Register argsort with NVFuser
 register_supported(prims.argsort, argsort_transform, _argsort_check_)
 
+
+def _grouped_mm_check(
+    a: TensorProxy,
+    b: TensorProxy,
+    offs: TensorProxy,
+) -> bool:
+    if not are_supported_tensors(a, b, offs):
+        return False
+
+    return nvfuser_version() >= LooseVersion("0.2.28")
+
+
+def _grouped_mm_transform(
+    a: TensorProxy,
+    b: TensorProxy,
+    offs: TensorProxy,
+    bias: TensorProxy = None,
```

Comment:
Since `prims._grouped_mm` doesn't have `bias` and `out_dtype` argument, I don't think the transform needs them as well.